Build fails because of pip exception

```
Preparing to unpack .../libyaml-dev_0.2.2-1_amd64.deb ...
Unpacking libyaml-dev:amd64 (0.2.2-1) ...
Setting up libyaml-dev:amd64 (0.2.2-1) ...
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.4/x64/bin/pip", line 5, in <module>
    from pip._internal.cli.main import main
  File "/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pip/_internal/cli/main.py", line 8, in <module>
    from pip._internal.cli.autocompletion import autocomplete
  File "/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pip/_internal/cli/autocompletion.py", line 9, in <module>
    from pip._internal.cli.main_parser import create_main_parser
  File "/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pip/_internal/cli/main_parser.py", line 7, in <module>
    from pip._internal.cli import cmdoptions
  File "/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pip/_internal/cli/cmdoptions.py", line 22, in <module>
    from pip._internal.cli.progress_bars import BAR_TYPES
  File "/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pip/_internal/cli/progress_bars.py", line 9, in <module>
    from pip._internal.utils.logging import get_indentation
  File "/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pip/_internal/utils/logging.py", line 14, in <module>
    from pip._internal.utils.misc import ensure_dir
  File "/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pip/_internal/utils/misc.py", line 29, in <module>
    from pip._internal.locations import get_major_minor_version, site_packages, user_site
  File "/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pip/_internal/locations/__init__.py", line 9, in <module>
    from . import _distutils, _sysconfig
  File "/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pip/_internal/locations/_sysconfig.py", line 8, in <module>
    from pip._internal.exceptions import InvalidSchemeCombination, UserInstallationInvalid
ImportError: cannot import name 'InvalidSchemeCombination' from 'pip._internal.exceptions' (/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pip/_internal/exceptions.py)
Error: Process completed with exit code 1.
0s
```

Probably related to https://github.com/pypa/pip/issues/9880

--- a/.github/workflows/release-staging.yaml
+++ b/.github/workflows/release-staging.yaml
@@ -132,10 +132,10 @@ jobs:
         with:
           python-version: '3.x'
 
-#      - uses: actions/cache@v2
-#        with:
-#          path: ${{ env.pythonLocation }}
-#          key: ${{ env.pythonLocation }}
+      - uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}
 
       - name: Installing Grow
         run: |

